src/crawlee/basic_crawler/_basic_crawler.py

```diff
@@ -336,12 +347,14 @@ async def run(
         *,
         purge_request_queue: bool = True,
     ) -> FinalStatistics:
-        """Run the crawler until all requests are processed.
+        """Runs the crawler until all requests are processed. Optionally purges the request queue for fresh runs.
 
         Args:
-            requests: The requests to be enqueued before the crawler starts
-            purge_request_queue: If this is `True` and the crawler is not being run for the first time, the default
-                request queue will be purged
+            requests: A list of requests to enqueue before the crawl starts.
+            purge_request_queue: Whether to clear the request queue before starting the crawler run.
+
+        Returns:
+            FinalStatistics: Contains the final statistics about the crawler run.
```

Comment:
Do not include types in the docstrings (`FinalStatistics:`), as we have type annotations for that.